sql/walrus--0.1.sql

```diff
@@ -113,7 +109,6 @@ create trigger tr_check_filters
 
 
 grant all on realtime.subscription to postgres;
```

Comment:
@olirice we can take this out since `postgres` was responsible for create the schema and table.

Reply:
fixed in 96bb612